packages/quantification/src/__tests__/validations.test.ts

```diff
@@ -0,0 +1,334 @@
+import type { Output } from '@nori-dot-com/ggit';
+
+import { SOIL_METRICS_RESULTS_NAME_SUFFIX } from '../constants';
+import { validateParsedModelRunsData } from '../validations';
+
+import { PARSED_SOIL_METRICS_OUTPUT } from './test-fixtures/parsed-model-runs';
+
+// Solving some type uncertainty with model runs being an object or an array
```

Comment:
This could be a typeguard instead. Alternatively I'm sure the type can be adjusted to account for this instead.